VogtPayroll/Employee.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll
+{
+    class Employee
+    {
+        private string _name;
+        private int _totalWorkingHours;
+        private decimal _payrate;
+
+        public Employee(string name, int totalWorkingHours, decimal payRate)
+        {
+            _name = name;
+            _totalWorkingHours = totalWorkingHours;
+            _payrate = payRate;
+        }
+
+        public Employee GetEmpInfo()
+        {
+            // allow user to enter employee name, total working hours and payrate
+
+            Console.WriteLine("What is the employee's name?");
```

Comment:
Move this entire function to Program.cs

Reply:
Have your Main() function call GetEmployeeInfo. Then use the response to then call PrintEmpInfo.

For example:

```c#
        static void Main(string[] args)
        {
            var employee = GetEmpInfo();
            employee.PrintempInfo();
        }
```